python/nammu/view/NammuView.py

```diff
@@ -91,9 +92,24 @@ def addCenterPane(self, atfAreaView, consoleView):
         # Make console's high remain smaller compared to edit area
         splitPane.setResizeWeight(0.9)
 
+    def set_title(self, unsaved=False):
+        """
+        Set the title bar to the base name of the currently open file.
+        If `unsaved` is `True`, prepend "(*) " to the file name.
+        """
+        if self.controller.currentFilename is None:
+            filename = "<New File>"
+        else:
+            filename = os.path.basename(self.controller.currentFilename)
+        if unsaved:
+            prefix = "(*) "
+        else:
+            prefix = ""
```

Comment:
These lines could be written a bit more briefly as:
```suggestion
            prefix =  "(*) " if unsaved else ""
```